chains/blastsepolia/logo.svg

```diff
@@ -0,0 +1,5 @@
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg" viewBox="50 50 200 200" preserveAspectRatio="xMidYMid meet">
+  <g transform="translate(0.000000,300.000000) scale(0.050000,-0.050000)" fill="#FCFC03" stroke="black" stroke-width="40">
+    <path d="M1394 4064 l-316 -234 1645 -5 c905 -3 1652 -12 1661 -21 9 -9 -23 -132 -71 -275 l-87 -259 -671 -5 -672 -6 -61 -184 c-34 -102 -62 -192 -62 -200 0 -8 297 -15 660 -15 531 0 660 -5 660 -26 0 -15 -45 -168 -101 -340 l-100 -314 -940 0 c-630 0 -939 7 -939 21 0 15 295 949 371 1175 7 22 -374 324 -410 324 -11 0 -641 -1951 -641 -1986 0 -8 540 -14 1200 -14 l1200 0 310 155 311 155 113 345 c62 189 109 348 105 351 -5 4 -95 69 -199 145 -105 77 -193 147 -197 158 -4 10 131 87 299 171 l306 154 108 324 c60 179 105 327 101 330 -4 4 -101 75 -216 159 l-209 153 -1421 -1 -1421 -1 -316 -234z"/>
+  </g>
+</svg>
```

Comment:
Hey @timertimertimer thank you for your PR to the Hyperlane Registry! Can you please crop and minify the SVG file to conform to the standard used in the repo? You can use these tools to do it:
- crop: https://svgcrop.com/
- minify: https://jakearchibald.github.io/svgomg/

Reply:
couldn't crop this svg logo, so took another one from blast brand kit